bin/start.sh

```diff
@@ -61,6 +61,12 @@ echo "source /opt/rh/rh-ruby23/enable" > /home/git/.bashrc
 ## Map PV repositories
 ln -sf $GIT_REPO_ROOT/repositories /home/git/repositories
 
+# Creating a pipe so that gitlab-shell processes can write into something that does not increase in size
+mkfifo /home/git/gitlab-shell/gitlab-shell.log
```

Comment:
Yes, will add a message for starting and exiting a tail. Thanks.

The gitlab-shell logging output looks very different to the sshd output so it is already easy to differentiate.